dashboard/app/views/home/_homepage.html.haml

```diff
@@ -13,3 +13,21 @@
 .clear{style: "clear: both"}
 
 #homepage-container
+
+#admin-extras
+  -if current_user.admin?
+    %a{href: CDO.code_org_url('/ops-dashboard#/districts'), style: 'text-decoration: none;'}
+      %h2
```

Comment:
This affects such a small percentage of users and it's low priority compared to so many other things for this launch, I don't want to focus on redesigning it at this point. It's supposed to go away completely or not be used by anyone anymore with all of the PLC updates that have been going out. Plus, I want it to be easily recognizable for those looking for it, especially since it jumped to the bottom of the page.